go.mod

```diff
@@ -3,11 +3,14 @@ module github.com/tron-us/go-btfs-common
 go 1.13
 
 require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/protobuf v1.3.2
 	github.com/libp2p/go-libp2p-core v0.0.6
 	github.com/stretchr/testify v1.3.0
 	github.com/tron-us/go-common/v2 v2.0.4
+	github.com/tron-us/protobuf v1.3.4
```

Comment:
v1.3.4 is based on a stable release of gogo proto right? Please document in that repo which release it is based on.. in case we need to upgrade in the future

Reply:
make sense, will do